pkg/ingester/active_series_test.go

```diff
@@ -116,7 +123,7 @@ func benchmarkActiveSeriesConcurrencySingleSeries(b *testing.B, goroutines int)
 
 			for ix := 0; ix < max; ix++ {
 				now = now.Add(time.Duration(ix) * time.Millisecond)
-				c.UpdateSeries(series, now, copyFn)
+				c.UpdateSeries(series, FromLabelAdaptersToLabels(series).Hash(), now, copyFn)
```

Comment:
Umm, can we move the hash calculation outside the benchmark loop?

Reply:
it seems should be.

Reply:
yes, it should be moved out.